infra/terraform/container-definitions-thumbor.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "thumbor",
+    "image": "wellcome/wellcomecollection-thumbor:test",
+    "cpu": 512,
+    "memory": 1000,
```

Comment:
[I have the count of tasks set to 1](https://github.com/wellcometrust/wellcomecollection.org/pull/871/files#diff-2da4d8e5604afe8134b645552b93f62dR61) for testing - so would need to either up the instances, or lower the momory.

I just want to get a gist for how much memory etc image processing has.